.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "singleQuote": true,
+    "trailingComma": "all",
+    "parser": "flow"
+}
```

Comment:
Add EOF here. Rest looks good to me :+1:

Reply:
Done thanks for figuring that out.